iis-workflows/iis-workflows-transformers/src/main/resources/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/oozie_app/workflow.xml

```diff
@@ -0,0 +1,43 @@
+<workflow-app xmlns="uri:oozie:workflow:0.5" name="transformers_citationmatching_direct (spark)">
+	
+	<parameters>
+		<property>
+			<name>input</name>
+			<description>input extracted document metadata merged with original</description>
+		</property>
+		<property>
+			<name>output</name>
+			<description>output direct citationmatching metadata</description>
+		</property>
+	</parameters>
+	 
+    <start to="transformer"/>
+    
+    <action name="transformer">
+        <spark xmlns="uri:oozie:spark-action:0.1">
+            <job-tracker>${jobTracker}</job-tracker>
+            <name-node>${nameNode}</name-node>
+			<!-- The data generated by this node is deleted in this section -->
+			<prepare>
+				<delete path="${nameNode}${output}" />
+			</prepare>
+            
+            <master>yarn-cluster</master>
+            <mode>cluster</mode>
+            <name>transformer_citationmatching_direct</name>
```

Comment:
I wonder if citationmatching_direct_transformer wouldn't be better? First the module, the the specific node.